README.rst

```diff
@@ -40,14 +40,57 @@ USAGE
 
 If you want to learn more about coala-bears, its functionality and its usage,
 please take a look at our
-`tutorial <http://coala.readthedocs.org/en/latest/Users/Tutorials/Writing_Bears.html#what-is-a-bear>`__.
+`tutorial <http://coala.readthedocs.org/en/latest/index.html>`__.
 
 ABOUT
 =====
 
-coala-bears is a Python package containing all the bears that is used by coala.
-
-Read more at our `documentation <http://coala.readthedocs.org/en/latest/Users/Tutorials/Writing_Bears.html#guide-to-write-a-bear>`__.
+coala-bears is a Python package containing all the bears that are officially
+supported by coala. It features more than **65 bears** covering **32 languages**.
+You can see all of them `here <https://github.com/coala-analyzer/coala/wiki/Bears>`_.
```

Comment:
When one changes, another should be changed. Why have both when can have one? Less maintenance is needed. Maybe, just a link to another would suffice instead of exactly the same content?